lib/python/picongpu/picmi/grid.py

```diff
@@ -83,9 +89,9 @@ def get_as_pypicongpu(self):
             raise ValueError("Z: boundary condition not supported")
 
         g = grid.Grid3D()
-        g.cell_size_x_si = (self.upper_bound[0] - self.lower_bound[0]) / self.number_of_cells[0]
-        g.cell_size_y_si = (self.upper_bound[1] - self.lower_bound[1]) / self.number_of_cells[1]
-        g.cell_size_z_si = (self.upper_bound[2] - self.lower_bound[2]) / self.number_of_cells[2]
+        g.cell_size_x_si = self.get_cell_size()[0]
+        g.cell_size_y_si = self.get_cell_size()[1]
+        g.cell_size_z_si = self.get_cell_size()[2]
```

Comment:
```suggestion
        g.cell_size_x_si, g.cell_size_y_si, g.cell_size_z_si = self.get_cell_size()
```
Not really important but this seems nicer to me.